File: iniciante/python/1184.py
# -*- coding: utf-8 -*-

O = input()
M = [[None for x in range(12)] for x in range(12)]
for i in range(len(M)):
    for j in range(len(M)):
        M[i][j] = float(input())

def funcao_soma(M):
    Soma = 0
    cont_elem = 0
    for i in range(len(M)):
        for j in range(len(M)):
            if j < i:
                Soma += M[i][j]
                cont_elem += 1
    return Soma, cont_elem
    
if O == 'S':
    S, num_elem = funcao_soma(M)
    print("%.1f" %S)

if O == 'M':
    S, num_elem = funcao_soma(M)
    media = S / num_elem
    print("%.1f" %media)

File: iniciante/c/1184.c
#include <stdio.h>
 
int main() {

	float M[12][12], soma, media;
	int i, j, contagem, diagonal;
	char O;

	scanf("%c", &O);
	for (i=0; i < 12;i++){
		for (j=0; j<12; j++){
			scanf("%f", &M[i][j]);
		}
	}
	soma = 0;
	contagem = 0;
	if (O == 'S'){
		for (i =0; i<12;i++){
			diagonal = i;
			for (j=0;j<12; j++){
				if (j < diagonal){
					soma += M[i][j];
					contagem += 1;
				}
			}
		}
		printf("%.1f\n", soma);
	}
	if (O == 'M'){
		for (i =0; i<12;i++){
			diagonal = i;
			for (j=0;j<12; j++){
				if (j < diagonal){
					soma += M[i][j];
					contagem += 1;
				}
			}
		}
		media = soma / contagem;
		printf("%.1f\n", media);
	}


    return 0;
}